Cython/Compiler/TypeSlots.py

```diff
@@ -1004,7 +1017,9 @@ def __init__(self, old_binops):
             MethodSlot(unaryfunc, "am_await", "__await__", method_name_to_slot),
             MethodSlot(unaryfunc, "am_aiter", "__aiter__", method_name_to_slot),
             MethodSlot(unaryfunc, "am_anext", "__anext__", method_name_to_slot),
-            EmptySlot("am_send", ifdef="PY_VERSION_HEX >= 0x030A00A3"),
+            # TODO: need to generate an appropriate C wrapper function in order to map .send() to the async slot.
+            # TODO: should we really map any .send() method to an async slot?
```

Comment:
This could do with testing I think. It isn't obvious to me whether this breaks every class with a non-generator `send` method, or if it's harmless.

Reply:
I removed this part. Just because a method is called `.send()` does not mean it has generator.send() semantics.